MobileCenterPush/MobileCenterPushTests/Model/MSMockPushDelegate.h

```diff
@@ -0,0 +1,9 @@
+/*
```

Comment:
This is one time use only. Why don't you move the declaration and definition of the Mock class to MobileCenterPushTests.m?